guides/common/attributes.adoc

```diff
@@ -5,13 +5,13 @@
 // Versions used in text and code
 :ProjectVersion: 2.3
 :KatelloVersion: 3.18
-:TargetVersion: 6.8
-:TargetVersionMaintainUpgrade: 6.8
+:TargetVersion: 6.9 Beta
```

Comment:
Refer to the attributes in the 6.8-beta branch downstream for the values. The values there are the following:
:TargetVersion: 6.8-beta
:ProductVersion: 6.8-beta
:ProductVersionRepoTitle: Beta

The ProductVersion attribute is used in links, it cannot have spaces there.

Reply:
Ack